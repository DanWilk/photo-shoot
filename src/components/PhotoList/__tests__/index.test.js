import React from 'react';
import {render, cleanup, getByTestId} from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import PhotoList from '..'

const category = {category: 'portraits'}

afterEach(cleanup);

describe(('Photolist is rendering'), () => {
    it('renders', () => {
        render(<PhotoList
            category = {category}
        />)
    })

    it(('matches snapshop'), () => {
        const { asFragment } = render(<PhotoList 
            category = {category}
        />)
        expect(asFragment).toMatchSnapshot()
    })
})