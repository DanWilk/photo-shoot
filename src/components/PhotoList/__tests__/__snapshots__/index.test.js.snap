// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photolist is rendering matches snapshop 1`] = `[Function]`;
